tests/cql/CqlListOperatorsTest.xml

```diff
@@ -527,7 +547,7 @@
 			<output>6</output>
 		</test>
 		<test name="LengthNullList">
-			<expression>Length(null as String)</expression>
+			<expression>Length(null as List&lt;Any&gt;)</expression>
```

Comment:
Since this is in the lists tests, I assume you mean for `null` to be cast as a list.